armi/reactor/blocks.py

```diff
@@ -1430,63 +1435,6 @@ def updateComponentDims(self):
             except NotImplementedError:
                 runLog.warning("{0} has no updatedDims method -- skipping".format(c))
 
-    def breakFuelComponentsIntoIndividuals(self):
```

Comment:
I like removing this, but just a head's up. I see this used downstream in certain neutronics and depletion codes.

If you've already got that handled, great. I'm just pointing it out, "just in case".

Reply:
Yep! It's all handed.

Reply:
I do agree we shouldn't use it and i added details of why
![image](https://github.com/user-attachments/assets/371711f6-63f6-4a55-89f5-b85160eca592)

part of me wonders if we really need to delete though. i could think of some corner case where it might be helpful....